### ISSUE TITLE
The execution time of the interface

### ISSUE THREAD

**Issue by JiaWenqi** [ayoungprogrammer/nlquery#9]
Is the execution time of the interface a little slow?
Which part of the interface executes slowly?

**Comment by ayoungprogrammer** [comment]
Yes the interface is a bit slow. This is a prototype not a production ready model. There are multiple calls to wikidata/dbpedia.
